Owncloud iOs Client/Login/Login/WebLoginViewController.swift

```diff
@@ -40,7 +39,10 @@ import Foundation
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        let cancelButton = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(self.cancelBarButtonPressed))
```

Comment:
Add the button in the storyboard and unwind with the segue: unwindToMainLoginViewWithSegue